docs/Intro/welcome.md

```diff
@@ -0,0 +1,38 @@
+# Welcome to Hyperledger fabric
+
+The Linux Foundation Hyperledger Project(1) is the result of collaboration between project members(2) 
+who share the common goal of designing a business-ready, open-source standard for blockchain networks. 
+Extensive research, planning and implementation by this community has led to the Linux Foundation adopting 
+Hyperledger fabric as its open source standard for blockchain network architecture.
+
+The Hyperledger fabric blockchain standard is a base architecture, with built-in functionality, security 
```

Comment:
this work is not a standards development effort. While we may hope that fabric reaches de facto standard status in terms of market acceptance and adoption, it is not a standard.